odl/solvers/advanced/proximal_operators.py

```diff
@@ -31,7 +31,7 @@
 from builtins import super
 
 import numpy as np
-import scipy.special
+import scipy
```

Comment:
Anyway, it seems you need to add `scipy.special` to the mock list. But you should still do `import scipy.special` up top, and then call `scipy.special.lambertw` later on, since this is more clear to readers. 1000 lines into a file it's hard to remember what was imported.